tests/behave/features/helpers/cla_public.py

```diff
@@ -0,0 +1,10 @@
+import os
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+
+def get_cla_public_url(context, path="start"):
```

Comment:
This function name reads to me like it will just return that url, which is a pattern of function name and behaviour that is pretty established. I wasn't expecting it to do a GET request (although knowing that it does, I understand why you called it what you did)

Is there another verb we could use in the function name to make the behaviour clearer? Something like "go to"?

Reply:
Changes this to `request_cla_public_path`